libraries/AP_ExternalControl/AP_ExternalControl.h

```diff
@@ -0,0 +1,28 @@
+/*
+  external control library for MAVLink, DDS and scripting
+ */
+
+#pragma once
+
+#include <AP_Math/AP_Math.h>
+
+class AP_ExternalControl {
+public:
+    /*
+      set linear velocity and yaw rate. Pass NaN for yaw_rate_rads to not control yaw
+      velocity is in earth frame, NED, m/s
+     */
```

Comment:
Should also say the frame convention for yaw rate? Even @lthall has realized the mavlink API's are not quite cut and dry. Although I don't recommend reading too much of this thread, this comment is useful. 
https://github.com/mavlink/mavlink/pull/2013#issuecomment-1612706250

Reply:
Yeh, I am going to put a PR in to fix this mistake. Once I saw it I wonder why I missed it the first time.

Reply:
Nice. Please tag me, so we can follow the same convention here.